### PR TITLE
de_DE Fixes and typos

### PR DESCRIPTION
**Pull request by SteWers**
## Description:
Fixes and typos

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works with Tasmota core ESP32 V.2.0.14
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

File: tasmota/language/de_DE.h [hunk 1]
```diff
@@ -526,7 +526,7 @@
 // xdrv_89_dali.ino
 #define D_SENSOR_DALI_RX                  "Dali RX"
 #define D_SENSOR_DALI_TX                  "Dali TX"
-#define D_CONFIGURE_DALI                  "Konfig․ DALI"
+#define D_CONFIGURE_DALI                  "DALI Einstellungen"
```
Comment (Noschvie) [inline tasmota/language/de_DE.h:529]: DALI-Einstellungen same as KNX-Einstellungen?

Comment (SteWers) [reply tasmota/language/de_DE.h:529]: Do you mean the hyphen?

Comment (Noschvie) [reply tasmota/language/de_DE.h:529]: yes, with or without the hyphen.

Comment (SteWers) [reply tasmota/language/de_DE.h]: Thanks for the hint.

**Pull request merged**